.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description
+<!--- 
+- a short summary of the pull request (PR)
+- acceptance criteria that you did not cover and why you did not cover these
+- open questions
+-->
+
+## Ticket Number
+<!--- Number of the ticket of this PR. -->
+
+# Changes
+
+## Added
+<!--- For new features. -->
+
+## Changes
+<!--- For changes in existing functionality. -->
+
+## Deprecated
+<!--- For soon-to-be removed features. -->
+
+## Removed
+<!--- For now removed features. -->
+
+## Fixed
+<!--- For any bug fixes. -->
+
+## Security
+<!--- In case of vulnerabilities. --> 
```

Comment:
this is for vulnerabilities i fixed or that i knowingly introduced? if the former, why differentiate that from `## Fixed`?

Reply:
This section is for fixed vulnerabilities, I clarified that in 407c280bf216800cc88e378b9abeb676efc2a43b
Regarding the difference: I am quite certain that organizations handle these kind of changes differently. E.g. security fixes are backported and bugfixes are not. Hence, I would like to keep it.